### PR TITLE
primary layouts

### PR DESCRIPTION
**Pull request by Khoacannotcode**
## Related Tickets:
- [45191](https://edu-redmine.sun-asterisk.vn/issues/45191)

## WHAT this PR does?
- Add primary layouts for admin static pages

## Notes Impacted Areas:
- N/A.

## Attachments:
image
imageimage
image
image

### CODE REVIEW

**Comment by Khoacannotcode** [comment]
ready

File: app/Http/Controllers/Admin/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function dashboard()
+    {
+        return view($this->path_to_view . 'dashboard', [
+            'title' => 'Meranda | Dashboard'
```
Comment (ghost) [inline app/Http/Controllers/Admin/AdminController.php]: cái này là gì vậy?

File: resources/views/admin/layouts/topbar.blade.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        <div class="dropdown-menu dropdown-menu-right shadow animated--grow-in"
+            aria-labelledby="userDropdown">
+            <a class="dropdown-item" href={{ route('admin.view_profile') }}>
+                <i class="fas fa-user fa-sm fa-fw mr-2 text-gray-400"></i>
+                Profile
+            </a>
+            <div class="dropdown-divider"></div>
+            <a class="dropdown-item" href="#" data-toggle="modal" data-target="#logoutModal">
+                <i class="fas fa-sign-out-alt fa-sm fa-fw mr-2 text-gray-400"></i>
+                Logout
```
Comment (ghost) [inline resources/views/admin/layouts/topbar.blade.php]: localization

File: app/Http/Controllers/Admin/AdminController.php (hunk 1 above)
Comment (Khoacannotcode) [reply app/Http/Controllers/Admin/AdminController.php]: Dạ em đưa title qua view anh ạ

**Comment by Khoacannotcode** [comment]
ready

**Pull request merged**